planetiler-core/src/main/java/com/onthegomap/planetiler/geo/DouglasPeuckerSimplifier.java

```diff
@@ -38,7 +38,7 @@ public class DouglasPeuckerSimplifier {
    * @return the simplified geometry
    */
   public static Geometry simplify(Geometry geom, double distanceTolerance) {
-    if (geom.isEmpty()) {
+    if (geom.isEmpty() || (distanceTolerance < 0.0)) {
```

Comment:
Should this be <= 0 so distance tolerance=0 has this same effect?

Reply:
Actually simplifying with tolerance=0 to remove colinear points is different from not simplifying at all so this is probably what we want. One nice ease-of-use improvement might be to make the arg so that you can do `--simplify-tolerance-at-max-zoom=false` or `=off` or something to turn it off instead of `=-1`

Reply:
The problem with `tolerance=0` is that it does still perform some simplification. I had initially filed #1092 before deciding that it was actually doing the right thing. My understanding is that if a node is EXACTLY on a line then `tolerance=0` will still remove it. There's certainly a difference in the output between using 0 and using -1.

Reply:
No problem at all, thanks for all your work on planetiler. It's proving invaluable for our port of a navigation app for the visually impaired. We're using the protomaps output from planetiler to drive both a maplibre graphical UI and an audio descriptive UI.